app/models/meal.rb

```diff
@@ -0,0 +1,7 @@
+class Meal < ApplicationRecord
+
+  belongs_to :week
+  validates :type, inclusion: { in: %w(eat_in eat_out),
+    message: "%{value} is not a valid size" }
```

Comment:
`is not a valid size` doesn't really make sense as a message here

just figured out what value is
`%{value} is not valid` would sound better I think